data/yam/agama/auto/lvm_sle.jsonnet

```diff
@@ -0,0 +1,36 @@
+{
+  "product": {
+    "id": "SLES_16.0"
+  },
+  "user": {
+    "fullName": "Bernhard M. Wiedemann",
+    "password": "$6$vYbbuJ9WMriFxGHY$gQ7shLw9ZBsRcPgo6/8KmfDvQ/lCqxW8/WnMoLCoWGdHO6Touush1nhegYfdBbXRpsQuy/FTZZeg7gQL50IbA/",
+    "hashedPassword": true,
+    "userName": "bernhard"
+  },
+  "root": {
+    "password": "$6$vYbbuJ9WMriFxGHY$gQ7shLw9ZBsRcPgo6/8KmfDvQ/lCqxW8/WnMoLCoWGdHO6Touush1nhegYfdBbXRpsQuy/FTZZeg7gQL50IbA/",
+    "hashedPassword": true
+  },
+  "storage": {
+    "guided": {
+      "target": {
+        "newLvmVg": [
+          "/dev/vda"
+        ]
+      }
+    }
+  },
+  "scripts": {
+    "post": [
+      {
+        "name": "enable root login",
+        "chroot": true,
+        "body": |||
+          #!/usr/bin/env bash
+          zypper rm openssh-server-config-rootlogin
```

Comment:
Can you please double confirm if this package is installed in target system?

Reply:
@chcao Please follow the PR https://github.com/os-autoinst/os-autoinst-distri-opensuse/pull/20885 to update jsonnet.